### DISCUSSION TITLE
Autocompletion for remote host.

### DISCUSSION THREAD

**Discussion by aliahadmd** [PowerShell/PowerShell#24000] [Enhancements]
Hi, I have been using remote server, but I can't get autocompletion/history like local machine. Is it possible to get auto history to my remote server from windows terminal?
Thanks.
Local windows terminal:
![image](https://github.com/PowerShell/PowerShell/assets/90032081/26380d4a-d21a-46e2-91cc-71061b395bee)

remote server through local windows terminal:
![image](https://github.com/PowerShell/PowerShell/assets/90032081/735fc7ef-a164-4fd6-b464-1c9e63e2abe9)

**Comment by rhubarb-geek-nz** [comment]
What type of remote server is it? How are you connecting to the remote server? That looks like connecting either with `wsl` or `ssh` with `bash` or similar running on the server.

**Comment by adardev** [comment]
That issue is because you went out powershell, once you init a remote server terminal, like in your case Im seeing is a bash terminal, the autocomplete feature of powershell is out, I recommend to you installing zsh or thousands of options out there, OhMyZsh, those are the ones I know, but good luck.
